src/DeleteCapableInterface.php

```diff
@@ -16,10 +18,18 @@
      *
      * @since [*next-version*]
      *
-     * @param LogicalExpressionInterface|null $condition An optional condition which, if specified, restricts the
-     *                                                   deletion to records that satisfy this condition.
+     * @param LogicalExpressionInterface|null   $condition An optional condition which, if specified, restricts the
+     *                                                     deletion to records that satisfy this condition.
+     * @param OrderInterface[]|Traversable|null $ordering  The ordering, as a list of OrderInterface instances.
+     * @param int|null                          $limit     The number of records to limit the query to.
```

Comment:
With normalization, `float`, `string` and `Stringable` should be accepted together with `int`.